File: Indie Effects Git/Assets/IndieEffects/JS Classes/Blur.js
#pragma strict
import IndieEffects;

@script RequireComponent (IndieEffects)
@script AddComponentMenu ("Indie Effects/Blur")
var fxRes : IndieEffects;

private var blurMat : Material;
var blurShader : Shader;
@range(0,5)
var blur : float;

function Start () {
	fxRes = GetComponent(IndieEffects);
	blurMat = new Material(blurShader);
}

function Update () {
	blurMat.SetTexture("_MainTex", fxRes.RT);
	blurMat.SetFloat("_Amount", blur);
}

function OnPostRender () {
	FullScreenQuad(blurMat);
}

File: Indie Effects Git/Assets/IndieEffects/CSharp Classes/Blur.cs
using UnityEngine;

[RequireComponent(typeof(IndieEffects))]
[AddComponentMenu("Indie Effects/Blur C#")]
public class Blur : MonoBehaviour
{
    public IndieEffects fxRes;

    private Material blurMat;
    public Shader blurShader;
    [Range(0,5)]
    public float blur;

    public void Start () {
	    fxRes = GetComponent<IndieEffects>();
	    blurMat = new Material(blurShader);
    }

    public void Update () {
	    blurMat.SetTexture("_MainTex", fxRes.RT);
	    blurMat.SetFloat("_Amount", blur);
    }

    public void OnPostRender () {
	    IndieEffects.FullScreenQuad(blurMat);
    }
}